packages/runtime/container-runtime/src/test/pendingStateManager.spec.ts

```diff
@@ -317,7 +320,9 @@ describe("Pending State Manager", () => {
 							0 /* batchStartCsn */,
 							false /* groupedBatch */,
 						),
+					// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
`IErrorBase` doesn't work here like in the test below?

Reply:
Good catch! Updated.